can_tools/scrapers/official/WI/wi_state.py

```diff
@@ -33,23 +32,480 @@ def normalize(self, data):
+            ),
+            "negative": CMU(
                 category="pcr_tests_negative",
-                measurement="new",
+                measurement="cumulative",
                 unit="unique_people",
             ),
             "pos_new": CMU(
                 category="pcr_tests_positive",
                 measurement="new",
                 unit="unique_people",
             ),
+            "neg_new": CMU(
+                category="pcr_tests_negative",
+                measurement="new",
+                unit="unique_people",
+            ),
             "test_new": CMU(
                 category="pcr_tests_total",
                 measurement="new",
                 unit="unique_people",
             ),
+            "deaths": CMU(category="deaths", measurement="cumulative", unit="people"),
+            "dth_new": CMU(category="deaths", measurement="new", unit="people"),
+            "hosp_yes": CMU(
+                category="hospital_beds_in_use_covid",
+                measurement="cumulative",
+                unit="people",
+            ),
+        }
+        out = (
+            df.melt(id_vars=["location"], value_vars=crename.keys())
+            .assign(
+                dt=self._retrieve_dt("US/Central"), vintage=self._retrieve_vintage()
+            )
+            .dropna()
+        )
+        out.loc[:, "value"] = pd.to_numeric(out["value"])
+
+        # Extract category information and add other variable context
+        out = self.extract_CMU(out, crename)
+
+        cols_to_keep = [
+            "vintage",
+            "dt",
+            "location",
+            "category",
+            "measurement",
+            "unit",
+            "age",
+            "race",
+            "ethnicity",
+            "sex",
+            "value",
+        ]
+
+        return out.loc[:, cols_to_keep]
+
+
+class WisconsinState(ArcGIS, DatasetBase):
+    """
+    Fetch state-level covid data from Wisconsin's ARCGIS dashboard
+    Includes demographic breakdowns
+    """
+
+    ARCGIS_ID = ""  # not applicable on this server
+    has_location = True
+    location_type = "state"
+    state_fips = int(us.states.lookup("Wisconsin").fips)
+    source = "https://www.dhs.wisconsin.gov/covid-19/data.htm"
+
+    # # override query URL since Wisconsin has a self-hosted ArcGIS server with a different URL pattern
+    def arcgis_query_url(
+        self, service="DHS_COVID19/COVID19_WI", sheet=3, srvid="server"
+    ):
+        out = f"https://dhsgis.wi.gov/{srvid}/rest/services/{service}/MapServer/{sheet}/query"
+
+        return out
+
+    def fetch(self):
+        return self.get_all_jsons("DHS_COVID19/COVID19_WI", 3, "server")
+
+    def normalize(self, data):
+        df = self.arcgis_jsons_to_df(data)
+        df.columns = [x.lower() for x in list(df)]
+        df["location"] = self.state_fips
+
+        crename = {
+            # #
+            # # totals across all sexes/ages/races/ethnicities
+            # #
+            "positive": CMU(
```

Comment:
@MattSidor @sglyon 

I'm pretty sure that I previously voted that we should label this as `pcr_tests_positive, cumulative, unique_people`, but now I'm wondering if this particular combination of variables should be given `cases, cumulative, people`... :thinking:

The nice thing being that then we'd have good information that was explicitly labeled as cases which is really what that combination of variables gives us. Keen to hear other thoughts on this though.

Reply:
I don't think I totally understand your last comment. Can you elaborate?

Reply:
For the demographic data, it seems like Wisconsin is reporting `pcr_tests_positive, cumulative, unique_people` as basically a fill-in for cases. We are getting `pcr_tests_positive, cumulative, unique_people` for each of the age/race/ethnicity/sex groups -- This feels more like they want us to think about these numbers as cases than testing data (they also report deaths for these groups).

My concrete proposal is that rather than store these as `pcr_tests_positive, cumulative, unique_people` that we call these `cases, cumulative, people`.

Reply:
sounds good to me. Thanks!

Reply:
Just to clarify @cc7768, do you want `pcr_tests_positive` changed to `cases` just for the demographic groups, or for all of the data in Wisconsin?

Reply:
I want to use `cases` instead of `pcr_tests_positive` only when we see the combination `pcr_tests_positive, cumulative, unique_people`

Reply:
Okay, will do. 👍